src/DependencyInjection/Configuration.php

```diff
@@ -60,6 +60,9 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
                                 ->useAttributeAsKey('target')
                                 ->prototype('scalar')->end()
                             ->end()
+                            ->scalarNode('default')
+                                ->info('Default values for target properties')
+                            ->end()
```

Comment:
I don't think this is the right place. It would mean that we have *one* default value for *all* populators defined in `properties`. I think we should add a default value *per property* instead.

Reply:
You're absolutely right; yesterday, I couldn't come up with a good idea to implement it. The suggested syntax doesn't seem to be valid in YAML, as far as I know. So, I began by tackling the simplest case and haven't finished it yet. Finally, a thought occurred to me: why not provide both options, allowing users to set a default value for all populators or for a specific property as needed?